knime_extension/src/nodes/spatialtool.py

```diff
@@ -461,6 +461,13 @@ class NearestJoinNode2:
 
     column_selection = ColumnSelection()
 
+    exclusive_equal = knext.BoolParameter(
+        label="Exclude the nearest geometries that are equal to the input geometry ",
+        description="""If checked, the nearest geometries that are equal to the input geometry will not be returned.""",
+        default_value=lambda v: True if v < knext.Version(1, 1, 0) else False,
+        since_version="1.1.0",
```

Comment:
Shouldn't the version be 1.3.0 in this line and the above one?

Reply:
Set since version to 1.3.0 in order to tell KNIME that this is a new parameter. This way existing workflows with use the default value.
Set the _default_value_ to false since it was always false and should be false by default.

Reply:
Revised. Be aware of the 1.3 version of 1.3 while testing